static/bidder-info/yieldmo.yaml

```diff
@@ -13,5 +13,5 @@ capabilities:
     - video
 userSync:
   redirect:
-    url: "https://ads.yieldmo.com/pbsync?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirectUri={{.RedirectURL}}"
+    url: "https://ads.yieldmo.com/pbsync?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redirectUri={{.RedirectURL}}"
```

Comment:
user sync url doesn't work for me. It says 400 bad request - 
![Screenshot 2024-02-05 at 4 31 34 PM](https://github.com/prebid/prebid-server/assets/8147531/39273ca8-09bb-4b12-b52b-652610af6607)

Here is an URL that I am trying -
`https://[ads.yieldmo.com](https://ads.yieldmo.com/pbsync?gdpr=&gdpr_consent=&us_privacy=&gpp=&gpp_sid=&redirectUri=http%3A%2F%2Flocalhost%3A8000%2Fsetuid%3Fbidder%3Dyieldmo%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Di%26uid%3D%24UID)/pbsync?gdpr=&gdpr_consent=&us_privacy=&gpp=&gpp_sid=&redirectUri=http%3A%2F%2Flocalhost%3A8000%2Fsetuid%3Fbidder%3Dyieldmo%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Di%26uid%3D%24UID`

Reply:
Your redirect uri is messed up. Try this: https://ads.yieldmo.com/pbsync?gdpr=&gdpr_consent=&us_privacy=&gpp=&gpp_sid=&redirectUri=https%3A%2F%2Fwww.cnn.com%3Fuid%3D%24UID

Reply:
@gargcreation1992 Can you please merge this?

Reply:
@gargcreation1992 @onkarvhanumante Any idea when this will be merged? It's blocking us. Thanks.